.github/workflows/publishing.yml

```
@@ -362,7 +365,7 @@ jobs:
           NGC_CLI_ORG: ${{ github.repository_owner }}
           NGC_CLI_TEAM: 'nightly'
         run: |
-          apt-get update && apt-get install -y --no-install-recommends wget
+          sudo apt-get update && sudo install -y --no-install-recommends wget
```

Was the `apt-get` removal intentional?
```suggestion
          sudo apt-get update && sudo apt-get install -y --no-install-recommends wget
```

---

No - thanks for catching that.